### PR TITLE
Update doc string for pred_contrib

### PR DESCRIPTION
**Pull request by slundberg**
See comments at the end of #1969

### CODE REVIEW

File: python-package/lightgbm/basic.py [hunk 1]
```diff
@@ -2261,9 +2261,11 @@ def predict(self, data, num_iteration=None,
 
             Note
             ----
-            If you want to get more explanation for your model's predictions using SHAP values
+            If you want to get more explanations for your model's predictions using SHAP values
             like SHAP interaction values,
-            you can install shap package (https://github.com/slundberg/shap).
+            you can install the shap package (https://github.com/slundberg/shap). Note that unlike
```
Comment (StrikerRUS) [inline python-package/lightgbm/basic.py]: Please, start new sentence from a new line. This makes docstrings more readable.

File: python-package/lightgbm/basic.py [hunk 2]
```diff
@@ -2261,9 +2261,11 @@ def predict(self, data, num_iteration=None,
...
+            the shap package, with pred_contrib we return a matrix with an extra column, where the
```
Comment (StrikerRUS) [inline python-package/lightgbm/basic.py]: Ah, and one more please: emphasize `pred_contrib`   with ``, like other params, e.g. feature_name here:
https://github.com/Microsoft/LightGBM/blob/beb35d567de899b140bd61e174ef3b9ef5fd0769/python-package/lightgbm/basic.py#L688

File: python-package/lightgbm/basic.py (hunk 1 above)
Comment (maximemerabet) [reply python-package/lightgbm/basic.py]: While we're at it, extremely minor format inconsistency in 

> shape = [n_samples] or shape = [n_samples, n_classes]

vs 

> shape = [n_samples, n_trees] or shape [n_samples, n_trees * n_classes]

Slightly OCD, I'll admit

Comment (slundberg) [reply python-package/lightgbm/basic.py]: @maximilianeber could you elaborate what you mean there?

Comment (maximilianeber) [reply python-package/lightgbm/basic.py]: @maximemerabet

Comment (StrikerRUS) [reply python-package/lightgbm/basic.py]: @maximilianeber Do you mean sign `=` after shape word?

Comment (slundberg) [reply python-package/lightgbm/basic.py]: @maximilianeber sorry for the auto-complete fail :) ...@maximemerabet just fixed the equals

Comment (maximemerabet) [reply python-package/lightgbm/basic.py]: Haha.

Yes was referring to the equals!

**Comment by slundberg** [comment]
No problem. Should be good to go now :)

**Comment by StrikerRUS** [comment]
@slundberg Oops! One more whitespace, which makes merge button disabled for me:

image

**Comment by slundberg** [comment]
My bad. Thought I had removed that one!

**Pull request merged**